test/components/views/messages/MPollBody-test.tsx

```diff
@@ -442,14 +460,14 @@ describe("MPollBody", () => {
         expect(body.html()).toBe("");
     });
 
-    it("renders nothing if poll has more than 20 answers", () => {
-        const answers = [...Array(21).keys()].map((i) => {
-            return { "id": `id${i}`, "org.matrix.msc1767.text": `Name ${i}` };
+    it("renders the first 20 answers if 21 were given", () => {
+        const answers = Array.from(Array(21).keys()).map((i) => {
+            return { "id": `id${i}`, [M_TEXT.name]: `Name ${i}` };
         });
         const votes = [];
         const ends = [];
         const body = newMPollBody(votes, ends, answers);
-        expect(body.html()).toBe("");
+        expect(body).toMatchSnapshot();
```

Comment:
This makes the test much harder to read and understand what is happening.  Is there some more explicit assertion we can make?

Reply:
possibly just a count of DOM elements, if I can figure out how to do that. Snapshot was just lazy and easy :p